src/version.js

```diff
@@ -1,6 +1,12 @@
-import React from 'react';
+/* eslint-disable no-var,object-shorthand */
 
-export const VERSION = React.version;
-export const REACT013 = VERSION.slice(0, 4) === '0.13';
-export const REACT014 = VERSION.slice(0, 4) === '0.14';
-export const REACT15 = VERSION.slice(0, 3) === '15.';
+var React = require('react');
+
+var VERSION = React.version;
+
+module.exports = {
```

Comment:
hmm, actually that's pretty inconvenient,  because the test files themselves import the source files from `src`, not from `build`, what do you suggest for that?